config/tizen/scripts/install_sdk.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
```

Comment:
Can we document the setup procedure somewhere, not just in the PR description?

Can we make sure this script itself is safe to run and gives sensible instructions when run without arguments?

Also, maybe we should encouraging install it into third_party/tizen_sdk ?

Clasically a problem arises when we put SDKs outside the source checkout... once we roll the SDK, having checkouts on different sides of the roll cannot work at the same time. I don't know how often this will update, but it's problematic especially for tools like 'git bisect'.

Reply:
Actually we will add documentations for build and flash, and execution for Tizen.
But for that, it is need to merge this PR in first. Based on this PR, we will add platform implementations. And will add documentations.

About the position of SDK, basically we refer on other platform's method such as "Android SDK setup", and don't put it inside of third_party. But also think about the method to use third_party/tizen_sdk/repo.
This method (third_party/tizen_sdk) is required the external github repo for git submodule. So after discussing internally, and getting more opnions from CHIP developrs, decide that method.

Reply:
Right now we have both approaches, e.g. Silicon Labs has the SDK in third_party so bisections works well there. Severity of the problem I mentioned depends on how frequently the SDK updates... at least with Android this is not too often.